benchmarks/dynamo/common.py

```diff
@@ -95,6 +95,18 @@
 
 log = logging.getLogger(__name__)
 
+# Enable OpenVINO Backend for benchmarking
+try:
+    import openvino.torch
```

Comment:
You shoudn't actually need to import a backend for it to work with `torch.compile(backend="openvino")` if you configure your package properly.  This is also true for benchmarking with `--backend=openvino`

This PR description explains how to setup an entrypoint for your package: https://github.com/pytorch/pytorch/pull/93873

This code will import your package:
https://github.com/pytorch/pytorch/blob/ea61c9cb299b6dfebc57dc9d8821c34321d568ab/torch/_dynamo/backends/registry.py#L98-L115

Reply:
how does it work with conda-forge where we cannot (?) specify torch entry points?